docs/3x/get_started.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+1. [Quick Samples](#quick-samples)
+
+2. [Feature Matrix](#feature-matrix)
+
+## Quick Samples
```

Comment:
We'd better have a quickstart for tf.

Reply:
I think we don't need much examples in get started, a simple one should be ok.